fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryManager.java

```diff
@@ -174,8 +174,8 @@ private Boolean mergePartition(@Nonnull final LuceneAnalyzerWrapper analyzerWrap
 
     private CompletableFuture<Integer> getNextPartitionInfo(final Tuple groupingKey, final AgilityContext agileContext,
```

Comment:
Probably worth renaming this to `getNextOlderPartitionInfo` too. If that's a lot of lines changed though, better as a followup.